style_guides/html_style_guide.md

```diff
@@ -27,4 +27,39 @@ If the element doesn't have children, add the closing tag on the same line as th
   attribute2="value2"
   attribute3="value3"
 ></div>
+```
+
+## Accessibility
+
+### Don't use the `title` attribute
+
+The `title` has no clear role within the accessibility standards. 
+[See the HTML5 spec](http://w3c.github.io/html/dom.html#the-title-attribute) for more information.
+
+To provide supplementary, descriptive information about a form control, button, link, or other element, use
+a tooltip component instead.
+
+Additional reading:
+
+* https://www.paciellogroup.com/blog/2010/11/using-the-html-title-attribute/
+* https://www.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/
+* https://www.deque.com/blog/text-links-practices-screen-readers/
+
+### Tooltips
```

Comment:
@BigFunger Here's an example of how we'd implement a tooltip with this guidance: https://codepen.io/cjcenizal/pen/aWqRVQ. Does this help?

I think we really need to have this accessibility information somewhere that's accessible (sorry) to everyone, so that we know where we want to go, even if we're not there yet. It's just a way of defining a goal first, and then finding a path there second.